### ISSUE TITLE
"Error wrapping InputStream in GZIPInputStream: java.io.EOFException" in compile:copy-resources

### ISSUE THREAD

**Issue by lambdaknight** [sbt/sbt#253]
I am attempting to use the "dist" command from Akka's SBT plugin and it fails with a non-deterministic error "Error wrapping InputStream in GZIPInputStream: java.io.EOFException".  Which projects fail with this error seems to be fairly random and it doesn't seem to happen when I turn parallel execution off, which leads me to suspect it's some sort of race condition happening with the parallel execution.  Below is the full stack trace:

```
Error wrapping InputStream in GZIPInputStream: java.io.EOFException
    at sbt.ErrorHandling$.translate(ErrorHandling.scala:10)
    at sbt.WrapUsing.open(Using.scala:36)
    at sbt.Using.apply(Using.scala:24)
    at sbt.IO$$anonfun$gzipFileIn$1.apply(IO.scala:596)
    at sbt.IO$$anonfun$gzipFileIn$1.apply(IO.scala:595)
    at sbt.Using.apply(Using.scala:25)
    at sbt.IO$.gzipFileIn(IO.scala:595)
    at sbt.Sync$.readUncaught(Sync.scala:88)
    at sbt.Sync$.readInfo(Sync.scala:84)
    at sbt.Sync$$anonfun$apply$1.apply(Sync.scala:29)
    at sbt.Sync$$anonfun$apply$1.apply(Sync.scala:23)
    at sbt.Defaults$$anonfun$copyResourcesTask$1.apply(Defaults.scala:509)
    at sbt.Defaults$$anonfun$copyResourcesTask$1.apply(Defaults.scala:505)
    at sbt.Scoped$$anonfun$hf5$1.apply(Structure.scala:461)
    at sbt.Scoped$$anonfun$hf5$1.apply(Structure.scala:461)
    at scala.Function1$$anonfun$compose$1.apply(Function1.scala:41)
    at sbt.Scoped$Reduced$$anonfun$combine$1$$anonfun$apply$11.apply(Structure.scala:280)
    at sbt.Scoped$Reduced$$anonfun$combine$1$$anonfun$apply$11.apply(Structure.scala:280)
    at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
    at sbt.std.Transform$$anon$5.work(System.scala:67)
    at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:221)
    at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:221)
    at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:13)
    at sbt.Execute.work(Execute.scala:227)
    at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:221)
    at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:221)
    at sbt.CompletionService$$anon$1$$anon$2.call(CompletionService.scala:26)
    at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
    at java.util.concurrent.FutureTask.run(FutureTask.java:138)
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:441)
    at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
    at java.util.concurrent.FutureTask.run(FutureTask.java:138)
    at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
    at java.lang.Thread.run(Thread.java:680)
Caused by: java.io.EOFException
    at java.util.zip.GZIPInputStream.readUByte(GZIPInputStream.java:249)
    at java.util.zip.GZIPInputStream.readUShort(GZIPInputStream.java:239)
    at java.util.zip.GZIPInputStream.readHeader(GZIPInputStream.java:142)
    at java.util.zip.GZIPInputStream.<init>(GZIPInputStream.java:58)
    at sbt.Using$$anonfun$gzipInputStream$1.apply(Using.scala:91)
    at sbt.Using$$anonfun$gzipInputStream$1.apply(Using.scala:91)
    at sbt.Using$$anon$1.openImpl(Using.scala:56)
    at sbt.WrapUsing$$anonfun$open$2.apply(Using.scala:36)
    at sbt.ErrorHandling$.translate(ErrorHandling.scala:9)
    at sbt.WrapUsing.open(Using.scala:36)
    at sbt.Using.apply(Using.scala:24)
    at sbt.IO$$anonfun$gzipFileIn$1.apply(IO.scala:596)
    at sbt.IO$$anonfun$gzipFileIn$1.apply(IO.scala:595)
    at sbt.Using.apply(Using.scala:25)
    at sbt.IO$.gzipFileIn(IO.scala:595)
    at sbt.Sync$.readUncaught(Sync.scala:88)
    at sbt.Sync$.readInfo(Sync.scala:84)
    at sbt.Sync$$anonfun$apply$1.apply(Sync.scala:29)
    at sbt.Sync$$anonfun$apply$1.apply(Sync.scala:23)
    at sbt.Defaults$$anonfun$copyResourcesTask$1.apply(Defaults.scala:509)
    at sbt.Defaults$$anonfun$copyResourcesTask$1.apply(Defaults.scala:505)
    at sbt.Scoped$$anonfun$hf5$1.apply(Structure.scala:461)
    at sbt.Scoped$$anonfun$hf5$1.apply(Structure.scala:461)
    at scala.Function1$$anonfun$compose$1.apply(Function1.scala:41)
    at sbt.Scoped$Reduced$$anonfun$combine$1$$anonfun$apply$11.apply(Structure.scala:280)
    at sbt.Scoped$Reduced$$anonfun$combine$1$$anonfun$apply$11.apply(Structure.scala:280)
    at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
    at sbt.std.Transform$$anon$5.work(System.scala:67)
    at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:221)
    at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:221)
    at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:13)
    at sbt.Execute.work(Execute.scala:227)
    at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:221)
    at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:221)
    at sbt.CompletionService$$anon$1$$anon$2.call(CompletionService.scala:26)
    at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
    at java.util.concurrent.FutureTask.run(FutureTask.java:138)
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:441)
    at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
    at java.util.concurrent.FutureTask.run(FutureTask.java:138)
    at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
    at java.lang.Thread.run(Thread.java:680)
```

**Comment by lambdaknight** [comment]
Did you close the right issue?  I still get this error and I'm not sure what your commit would do to fix the error.

**Comment by waywardmonkeys** [comment]
As a note, this is from sharing an output directory between multiple projects.

**Comment by sslavic** [comment]
Just experienced same error, sbt 0.13.7, single root project, scala 2.10.4, jdk 1.7. Command executed "sbt clean package dist". Resources filtered with xsbt-filter 0.4 and project packaged into java distribution archive with sbt-native-packager 1.0.0-M4.

```
build   24-Feb-2015 09:53:34    [0m[[0minfo[0m] [0mDone packaging.[0m
build   24-Feb-2015 09:53:34    Error wrapping InputStream in GZIPInputStream: java.io.EOFException
build   24-Feb-2015 09:53:34        at sbt.ErrorHandling$.translate(ErrorHandling.scala:10)
build   24-Feb-2015 09:53:34        at sbt.WrapUsing.open(Using.scala:34)
build   24-Feb-2015 09:53:34        at sbt.Using.apply(Using.scala:23)
build   24-Feb-2015 09:53:34        at sbt.IO$$anonfun$gzipFileIn$1.apply(IO.scala:810)
build   24-Feb-2015 09:53:34        at sbt.IO$$anonfun$gzipFileIn$1.apply(IO.scala:809)
build   24-Feb-2015 09:53:34        at sbt.Using.apply(Using.scala:24)
build   24-Feb-2015 09:53:34        at sbt.IO$.gzipFileIn(IO.scala:809)
build   24-Feb-2015 09:53:34        at sbt.Sync$.readUncaught(Sync.scala:88)
build   24-Feb-2015 09:53:34        at sbt.Sync$.readInfo(Sync.scala:84)
build   24-Feb-2015 09:53:34        at sbt.Sync$$anonfun$apply$1.apply(Sync.scala:28)
build   24-Feb-2015 09:53:34        at sbt.Sync$$anonfun$apply$1.apply(Sync.scala:22)
build   24-Feb-2015 09:53:34        at sbt.Defaults$$anonfun$copyResourcesTask$1.apply(Defaults.scala:840)
build   24-Feb-2015 09:53:34        at sbt.Defaults$$anonfun$copyResourcesTask$1.apply(Defaults.scala:836)
build   24-Feb-2015 09:53:34        at scala.Function4$$anonfun$tupled$1.apply(Function4.scala:35)
build   24-Feb-2015 09:53:34        at scala.Function4$$anonfun$tupled$1.apply(Function4.scala:34)
build   24-Feb-2015 09:53:34        at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
build   24-Feb-2015 09:53:34        at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
build   24-Feb-2015 09:53:34        at sbt.std.Transform$$anon$4.work(System.scala:63)
build   24-Feb-2015 09:53:34        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
build   24-Feb-2015 09:53:34        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
build   24-Feb-2015 09:53:34        at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
build   24-Feb-2015 09:53:34        at sbt.Execute.work(Execute.scala:235)
build   24-Feb-2015 09:53:34        at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
build   24-Feb-2015 09:53:34        at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
build   24-Feb-2015 09:53:34        at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
build   24-Feb-2015 09:53:34        at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
build   24-Feb-2015 09:53:34        at java.util.concurrent.FutureTask.run(FutureTask.java:262)
build   24-Feb-2015 09:53:34        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
build   24-Feb-2015 09:53:34        at java.util.concurrent.FutureTask.run(FutureTask.java:262)
build   24-Feb-2015 09:53:34        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
build   24-Feb-2015 09:53:34        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
build   24-Feb-2015 09:53:34        at java.lang.Thread.run(Thread.java:745)
build   24-Feb-2015 09:53:34    Caused by: java.io.EOFException
build   24-Feb-2015 09:53:34        at java.util.zip.GZIPInputStream.readUByte(GZIPInputStream.java:264)
build   24-Feb-2015 09:53:34        at java.util.zip.GZIPInputStream.readUShort(GZIPInputStream.java:254)
build   24-Feb-2015 09:53:34        at java.util.zip.GZIPInputStream.readHeader(GZIPInputStream.java:163)
build   24-Feb-2015 09:53:34        at java.util.zip.GZIPInputStream.<init>(GZIPInputStream.java:78)
build   24-Feb-2015 09:53:34        at sbt.Using$$anonfun$gzipInputStream$1.apply(Using.scala:84)
build   24-Feb-2015 09:53:34        at sbt.Using$$anonfun$gzipInputStream$1.apply(Using.scala:84)
build   24-Feb-2015 09:53:34        at sbt.Using$$anon$1.openImpl(Using.scala:51)
build   24-Feb-2015 09:53:34        at sbt.WrapUsing$$anonfun$open$2.apply(Using.scala:34)
build   24-Feb-2015 09:53:34        at sbt.ErrorHandling$.translate(ErrorHandling.scala:10)
build   24-Feb-2015 09:53:34        at sbt.WrapUsing.open(Using.scala:34)
build   24-Feb-2015 09:53:34        at sbt.Using.apply(Using.scala:23)
build   24-Feb-2015 09:53:34        at sbt.IO$$anonfun$gzipFileIn$1.apply(IO.scala:810)
build   24-Feb-2015 09:53:34        at sbt.IO$$anonfun$gzipFileIn$1.apply(IO.scala:809)
build   24-Feb-2015 09:53:34        at sbt.Using.apply(Using.scala:24)
build   24-Feb-2015 09:53:34        at sbt.IO$.gzipFileIn(IO.scala:809)
build   24-Feb-2015 09:53:34        at sbt.Sync$.readUncaught(Sync.scala:88)
build   24-Feb-2015 09:53:34        at sbt.Sync$.readInfo(Sync.scala:84)
build   24-Feb-2015 09:53:34        at sbt.Sync$$anonfun$apply$1.apply(Sync.scala:28)
build   24-Feb-2015 09:53:34        at sbt.Sync$$anonfun$apply$1.apply(Sync.scala:22)
build   24-Feb-2015 09:53:34        at sbt.Defaults$$anonfun$copyResourcesTask$1.apply(Defaults.scala:840)
build   24-Feb-2015 09:53:34        at sbt.Defaults$$anonfun$copyResourcesTask$1.apply(Defaults.scala:836)
build   24-Feb-2015 09:53:34        at scala.Function4$$anonfun$tupled$1.apply(Function4.scala:35)
build   24-Feb-2015 09:53:34        at scala.Function4$$anonfun$tupled$1.apply(Function4.scala:34)
build   24-Feb-2015 09:53:34        at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
build   24-Feb-2015 09:53:34        at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
build   24-Feb-2015 09:53:34        at sbt.std.Transform$$anon$4.work(System.scala:63)
build   24-Feb-2015 09:53:34        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
build   24-Feb-2015 09:53:34        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
build   24-Feb-2015 09:53:34        at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
build   24-Feb-2015 09:53:34        at sbt.Execute.work(Execute.scala:235)
build   24-Feb-2015 09:53:34        at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
build   24-Feb-2015 09:53:34        at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
build   24-Feb-2015 09:53:34        at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
build   24-Feb-2015 09:53:34        at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
build   24-Feb-2015 09:53:34        at java.util.concurrent.FutureTask.run(FutureTask.java:262)
build   24-Feb-2015 09:53:34        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
build   24-Feb-2015 09:53:34        at java.util.concurrent.FutureTask.run(FutureTask.java:262)
build   24-Feb-2015 09:53:34        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
build   24-Feb-2015 09:53:34        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
build   24-Feb-2015 09:53:34        at java.lang.Thread.run(Thread.java:745)
build   24-Feb-2015 09:53:34    [0m[[31merror[0m] [0m(compile:filterResources::[31mcopyResources[0m) Error wrapping InputStream in GZIPInputStream: java.io.EOFException[0m
```

**Comment by sslavic** [comment]
Please consider reopening this ticket, it happens too frequently in my build - I'm using xsbt-filter to filter some resources which are excluded from bin package but included in sbt-native-packager java application distribution package.

```
build   19-Apr-2015 18:09:00    Error wrapping InputStream in GZIPInputStream: java.io.EOFException
build   19-Apr-2015 18:09:00        at sbt.ErrorHandling$.translate(ErrorHandling.scala:10)
build   19-Apr-2015 18:09:00        at sbt.WrapUsing.open(Using.scala:34)
build   19-Apr-2015 18:09:00        at sbt.Using.apply(Using.scala:23)
build   19-Apr-2015 18:09:00        at sbt.IO$$anonfun$gzipFileIn$1.apply(IO.scala:810)
build   19-Apr-2015 18:09:00        at sbt.IO$$anonfun$gzipFileIn$1.apply(IO.scala:809)
build   19-Apr-2015 18:09:00        at sbt.Using.apply(Using.scala:24)
build   19-Apr-2015 18:09:00        at sbt.IO$.gzipFileIn(IO.scala:809)
build   19-Apr-2015 18:09:00        at sbt.Sync$.readUncaught(Sync.scala:88)
build   19-Apr-2015 18:09:00        at sbt.Sync$.readInfo(Sync.scala:84)
build   19-Apr-2015 18:09:00        at sbt.Sync$$anonfun$apply$1.apply(Sync.scala:28)
build   19-Apr-2015 18:09:00        at sbt.Sync$$anonfun$apply$1.apply(Sync.scala:22)
build   19-Apr-2015 18:09:00        at sbt.Defaults$$anonfun$copyResourcesTask$1.apply(Defaults.scala:840)
build   19-Apr-2015 18:09:00        at sbt.Defaults$$anonfun$copyResourcesTask$1.apply(Defaults.scala:836)
build   19-Apr-2015 18:09:00        at scala.Function4$$anonfun$tupled$1.apply(Function4.scala:35)
build   19-Apr-2015 18:09:00        at scala.Function4$$anonfun$tupled$1.apply(Function4.scala:34)
build   19-Apr-2015 18:09:00        at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
build   19-Apr-2015 18:09:00        at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
build   19-Apr-2015 18:09:00        at sbt.std.Transform$$anon$4.work(System.scala:63)
build   19-Apr-2015 18:09:00        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
build   19-Apr-2015 18:09:00        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
build   19-Apr-2015 18:09:00        at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
build   19-Apr-2015 18:09:00        at sbt.Execute.work(Execute.scala:235)
build   19-Apr-2015 18:09:00        at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
build   19-Apr-2015 18:09:00        at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
build   19-Apr-2015 18:09:00        at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
build   19-Apr-2015 18:09:00        at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
build   19-Apr-2015 18:09:00        at java.util.concurrent.FutureTask.run(FutureTask.java:262)
build   19-Apr-2015 18:09:00        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
build   19-Apr-2015 18:09:00        at java.util.concurrent.FutureTask.run(FutureTask.java:262)
build   19-Apr-2015 18:09:00        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
build   19-Apr-2015 18:09:00        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
build   19-Apr-2015 18:09:00        at java.lang.Thread.run(Thread.java:745)
build   19-Apr-2015 18:09:00    Caused by: java.io.EOFException
build   19-Apr-2015 18:09:00        at java.util.zip.GZIPInputStream.readUByte(GZIPInputStream.java:264)
build   19-Apr-2015 18:09:00        at java.util.zip.GZIPInputStream.readUShort(GZIPInputStream.java:254)
build   19-Apr-2015 18:09:00        at java.util.zip.GZIPInputStream.readHeader(GZIPInputStream.java:163)
build   19-Apr-2015 18:09:00        at java.util.zip.GZIPInputStream.<init>(GZIPInputStream.java:78)
build   19-Apr-2015 18:09:00        at sbt.Using$$anonfun$gzipInputStream$1.apply(Using.scala:84)
build   19-Apr-2015 18:09:00        at sbt.Using$$anonfun$gzipInputStream$1.apply(Using.scala:84)
build   19-Apr-2015 18:09:00        at sbt.Using$$anon$1.openImpl(Using.scala:51)
build   19-Apr-2015 18:09:00        at sbt.WrapUsing$$anonfun$open$2.apply(Using.scala:34)
build   19-Apr-2015 18:09:00        at sbt.ErrorHandling$.translate(ErrorHandling.scala:10)
build   19-Apr-2015 18:09:00        at sbt.WrapUsing.open(Using.scala:34)
build   19-Apr-2015 18:09:00        at sbt.Using.apply(Using.scala:23)
build   19-Apr-2015 18:09:00        at sbt.IO$$anonfun$gzipFileIn$1.apply(IO.scala:810)
build   19-Apr-2015 18:09:00        at sbt.IO$$anonfun$gzipFileIn$1.apply(IO.scala:809)
build   19-Apr-2015 18:09:00        at sbt.Using.apply(Using.scala:24)
build   19-Apr-2015 18:09:00        at sbt.IO$.gzipFileIn(IO.scala:809)
build   19-Apr-2015 18:09:00        at sbt.Sync$.readUncaught(Sync.scala:88)
build   19-Apr-2015 18:09:00        at sbt.Sync$.readInfo(Sync.scala:84)
build   19-Apr-2015 18:09:00        at sbt.Sync$$anonfun$apply$1.apply(Sync.scala:28)
build   19-Apr-2015 18:09:00        at sbt.Sync$$anonfun$apply$1.apply(Sync.scala:22)
build   19-Apr-2015 18:09:00        at sbt.Defaults$$anonfun$copyResourcesTask$1.apply(Defaults.scala:840)
build   19-Apr-2015 18:09:00        at sbt.Defaults$$anonfun$copyResourcesTask$1.apply(Defaults.scala:836)
build   19-Apr-2015 18:09:00        at scala.Function4$$anonfun$tupled$1.apply(Function4.scala:35)
build   19-Apr-2015 18:09:00        at scala.Function4$$anonfun$tupled$1.apply(Function4.scala:34)
build   19-Apr-2015 18:09:00        at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
build   19-Apr-2015 18:09:00        at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
build   19-Apr-2015 18:09:00        at sbt.std.Transform$$anon$4.work(System.scala:63)
build   19-Apr-2015 18:09:00        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
build   19-Apr-2015 18:09:00        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
build   19-Apr-2015 18:09:00        at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
build   19-Apr-2015 18:09:00        at sbt.Execute.work(Execute.scala:235)
build   19-Apr-2015 18:09:00        at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
build   19-Apr-2015 18:09:00        at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
build   19-Apr-2015 18:09:00        at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
build   19-Apr-2015 18:09:00        at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
build   19-Apr-2015 18:09:00        at java.util.concurrent.FutureTask.run(FutureTask.java:262)
build   19-Apr-2015 18:09:00        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
build   19-Apr-2015 18:09:00        at java.util.concurrent.FutureTask.run(FutureTask.java:262)
build   19-Apr-2015 18:09:00        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
build   19-Apr-2015 18:09:00        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
build   19-Apr-2015 18:09:00        at java.lang.Thread.run(Thread.java:745)
build   19-Apr-2015 18:09:00    [0m[[31merror[0m] [0m(foo/compile:[31mcopyResources[0m) Error wrapping InputStream in GZIPInputStream: java.io.EOFException[0m
build   19-Apr-2015 18:09:00    [0m[[31merror[0m] [0mTotal time: 37 s, completed Apr 19, 2015 6:09:00 PM[0m
simple  19-Apr-2015 18:09:01    Failing task since return code of [/bin/bash sbt clean dist] was 1 while expected 0
```

**Comment by iurisman** [comment]
@lambdaknight, why is this closed w/o explanation? It's 2017 and this is still an issue.

**Comment by lambdaknight** [comment]
I have no idea. It was closed in 2011 originally for unknown reasons.

**Comment by iurisman** [comment]
My bad — I should have asked @harrah. Mark, any comment comment?

**Comment by eed3si9n** [comment]
I ran into this while switching between sbt versions, so I fixed it in #3134.
Could you try 0.13.16-M1 if it's reproducible?
